src/components/useList/components/ListItemView/ListItemViewContent.tsx

```diff
@@ -57,7 +58,17 @@ export const ListItemViewContent = ({
     expanded,
     selected,
     title,
+    expandIconPosition = 'start',
+    renderExpandIcon: RenderExpandIcon = ListItemExpandIcon,
```

Comment:
What the case for custom render?

Reply:
if you wont custom icon view

Reply:
This prop called `render*` but it's not a function. Also it takes props of `ListItemExpandIcon`, so when I want custom, I should rely on this props?. Feels a bit overwhelming and hard to understand.

Reply:
this is exactly a render function with props. Here fased link to a `ListItemExpandIcon` withot calling. In most cases you don't need to use this prop - it's optional